docs/layouts/shortcodes/generated/expert_rest_section.html

```diff
@@ -20,6 +20,18 @@
             <td>Long</td>
             <td>The time in ms that the client waits for the leader address, e.g., Dispatcher or WebMonitorEndpoint</td>
         </tr>
+        <tr>
+            <td><h5>rest.cache.checkpoint-statistics.size</h5></td>
```

Comment:
Cn you justify why this config should go into the `rest.*` bucket and not the `web.*` bucket? Seems llike we have similar configs in the `web.*` section, for example `web.exception-history-size`

Reply:
In my mind, `rest.*` should be configs that are REST API relation, whereas `web.*` should control configs that are Flink dashboard related (e.g. `web.refresh-interval`. 

Decided to go with `rest.*` because this cache is a REST API concept